packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/components/StickyAdvancedSettings/index.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { GridItem } from '@strapi/design-system/Grid';
+import { Stack } from '@strapi/design-system/Stack';
+
+// 56px * 3 = height of header + top & bottom padding of main container
+// 24px = vertical space between FormApiTokenContainer & Permissions
+export const CalculatedHeightStack = styled(Stack)`
+  height: calc(100vh - 56px * 3 - 24px);
```

Comment:
Mmmh, I'm not convinced we should use the fixed `56px` here, because this might change if users are using a different font-size or zoomed in.

I haven't checked how it works, but could you think of a more generic way to solve that?

Reply:
@gu-stav  Thanks for the review, we can use value from `theme` to replace fixed values but now we realize that the sticky header doesn't have fixed value, it could be higher with a long enough title. Let's try to find a more generic way.

Reply:
@gu-stav We learnt from implementation of `HeaderLayout` from system design and made similar changes to resolve this issue. Could you please take a look?

Reply:
@gu-stav ^^!